sdk/servicebus/service-bus/src/serializers/queueResourceSerializer.ts

```diff
@@ -360,180 +372,45 @@ export interface InternalQueueOptions {
 }
 
 /**
- * @internal
- * @ignore
- * Represents all attributes of a queue entity
+ * Represents runtime info attributes of a queue entity
  */
-export interface QueueDetails {
+export interface QueueRuntimeInfo {
```

Comment:
Keep the `@internal` and `@ignore` tags on the `*RuntimeInfo` interfaces for this PR. They can be removed in the next PR when we add support for the runtime info methods

Reply:
Done

Reply:
Then why does it still appear in the api review markdown file....